action.yaml

```diff
@@ -56,6 +56,20 @@ runs:
     env:
       BUILDDIR: build
 
+  # Try to eliminate "Failed to dissect image: Connection timed out" errors from nspawn by compiling
+  # systemd-nspawn from v251 from source.
+  - name: Update systemd-nspawn
+    shell: bash
+    run: |
+      echo "deb-src http://archive.ubuntu.com/ubuntu/ $(lsb_release -cs) main restricted universe multiverse" | sudo tee -a /etc/apt/sources.list
+      sudo apt-get update
+      sudo apt-get build-dep systemd
+      git clone https://github.com/systemd/systemd-stable --branch v251.2 --depth=1
+      meson systemd-stable/build systemd-stable
+      ninja -C systemd-stable/build
```

Comment:
we should add a meson build target for nspawn, if it's not there already, to speed this up a bit and avoid to rebuild the world?

Reply:
Good point, this is available, let me post a PR to use it.